src/main.rs

```diff
@@ -143,27 +138,15 @@ fn main() {
         .parse::<u32>()
         .unwrap();
 
-    let (is_jailed, start_time_us, start_time_cpu_us) = if cmd_arguments.is_present("jailed") {
-        (
-            true,
-            Some(
-                cmd_arguments
-                    .value_of("start-time-us")
-                    .expect("'start-time-us' parameter expected when running jailed.")
-                    .parse::<u64>()
-                    .expect("'start-time-us' parameter expected to be of <u64> type."),
-            ),
-            Some(
-                cmd_arguments
-                    .value_of("start-time-cpu-us")
-                    .expect("'start-time-cpu-us' parameter expected when running jailed.")
-                    .parse::<u64>()
-                    .expect("'start-time-cpu-us' parameter expected to be of <u64> type."),
-            ),
-        )
-    } else {
-        (false, None, None)
-    };
+    let start_time_us = cmd_arguments.value_of("start-time-us").map(|s| {
```

Comment:
Why not use `clap::Arg::validator` to enforce argument validation?

Reply:
Tbh I didn't find a `u64` validator and was lazy to implement one. Even if we use a `validator`, we still need to do the `.map()` to get a `Option<u64>` from the `Option<string>`.